### PR TITLE
[v6r15] Fix dirac-rss-sync command to not call JobDB directly

### PR DESCRIPTION
**Pull request by atsareg**
Accessing ResourceStatusDB and JobDB through clients rather than directly.
Proper order in the script initialization and import statements

### CODE REVIEW

File: ResourceStatusSystem/scripts/dirac-rss-sync.py [hunk 1]
```diff
@@ -127,10 +135,9 @@ def initSites():
     Initializes Sites statuses taking their values from the "SiteMask" table of "JobDB" database.
   '''
 
-  jobClient = JobDB()
-  rssClient = ResourceStatusDB()
+  rssClient = ResourceStatusClient.ResourceStatusClient()
```
Comment (vipersec) [inline ResourceStatusSystem/scripts/dirac-rss-sync.py]: `ResourceStatusClient.ResourceStatusClient()` does not contain ResourceStatusDB's `addIfNotThere` function which is used after a few lines [here](https://github.com/atsareg/DIRAC/blob/32a6f5f310d42203e65bed28ded69defdc467998/ResourceStatusSystem/scripts/dirac-rss-sync.py#L151)

Comment (atsareg) [inline ResourceStatusSystem/scripts/dirac-rss-sync.py]: Ah, I see. I did not pay attention that this client is not inheriting the base Client class. Is there a special reason for that ? With this inheritance all the service methods are immediately available as if they implemented in the client. Instead, there is an internal redirection which is doing essentially the same thing.

**Comment by atsareg** [comment]
Not subject to this PR, but the ResourceStatusClient implementation must be reviewed. I think it can be ways simpler by making it inherit the Client base class.

File: ResourceStatusSystem/scripts/dirac-rss-sync.py [hunk 2]
```diff
@@ -141,7 +148,7 @@ def initSites():
     insert = { 'Status': elements[0], 'Reason': 'Synchronized', 'Name': site, 'DateEffective': elements[1], 'TokenExpiration': Datetime,
              'ElementType': 'Site', 'StatusType': 'all', 'LastCheckTime': None, 'TokenOwner': elements[2] }
 
-    result = rssClient.addIfNotThere(insert, table)
+    result = rssClient.addIfNotThereStatusElement( insert, table )
```
Comment (vipersec) [inline ResourceStatusSystem/scripts/dirac-rss-sync.py]: The `addIfNotThereStatusElement` gets string parameters and not dictionaries as far as I know.

File: ResourceStatusSystem/scripts/dirac-rss-sync.py (hunk 1 above)
Comment (vipersec) [inline ResourceStatusSystem/scripts/dirac-rss-sync.py]: Perhaps it should be changed in the near future.

Comment (atsareg) [inline ResourceStatusSystem/scripts/dirac-rss-sync.py]: Ok, I made a fix  following your suggestions, it works

**Pull request merged**